index.js

```diff
@@ -0,0 +1,80 @@
+const apiUrl = 'https://ghibliapi.herokuapp.com';
+let localMovieReviewStorage = {};
+
+document.addEventListener('DOMContentLoaded', () => {
+	seedSelectBox();
+	let selectBox = document.querySelector('.movie-titles');
+	selectBox.addEventListener('change', getMovieInfo);
+	let form = document.querySelector('.user-review');
+	form.addEventListener('submit', makeNewReview);
+});
+
+async function seedSelectBox(){
+	let selectBox = document.querySelector('.movie-titles');
+	let movies = await axios.get(`${apiUrl}/films`);
+	let defaultOption = document.createElement('option');
+	selectBox.appendChild(defaultOption);
+	for(let i = 0; i < movies.data.length; i++){
+		let option = document.createElement('option');
+		option.text = movies.data[i].title;
+		localMovieReviewStorage[movies.data[i].title] = [];
+		selectBox.appendChild(option);
+	}
+}
+
+async function getMovieInfo(){
+	let descriptionBoard = document.querySelector('.movie-description');
+	let child = descriptionBoard.firstElementChild;
+	//removes all child elements from the board
+	while(child){
+		descriptionBoard.removeChild(child);
+		child = descriptionBoard.firstElementChild;
+	}
+	let selectBox = document.querySelector('.movie-titles');
+	let movies = await axios.get(`${apiUrl}/films`);
```

Comment:
You could have stored the film `id` in the `optionElement.value` so that when the user picks an option `selectElement.value` will have the id. [Check this](https://www.w3schools.com/jsref/prop_select_value.asp) click `Try it yourself`. This would have prevented you from having to make another network request to `/films`. Instead you would make a request to `/films/<film-id>` which was our intent.